copy/copy.go

```diff
@@ -70,12 +70,11 @@ type copier struct {
 	progressInterval      time.Duration
 	progress              chan types.ProgressProperties
 	blobInfoCache         internalblobinfocache.BlobInfoCache2
-	copyInParallel        bool
 	compressionFormat     *compressiontypes.Algorithm // Compression algorithm to use, if the user explicitly requested one, or nil.
 	compressionLevel      *int
 	ociDecryptConfig      *encconfig.DecryptConfig
 	ociEncryptConfig      *encconfig.EncryptConfig
-	maxParallelDownloads  uint
+	copySemaphore         *semaphore.Weighted
```

Comment:
Non-blocking?

The `copySemaphore` name in `copier.copySemaphore *semaphore` adds no information over the parent and type, and the scope is now larger so there is more ambiguity.

`blobCopySemaphore`? `parallelBlobCopySemaphore`? Or just move something like the original comment here?

Reply:
Renamed to `concurrentBlobCopiesSemaphore`